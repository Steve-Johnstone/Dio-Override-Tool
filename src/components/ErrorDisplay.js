const ErrorDisplay = ({ error }) => {
	return (
		<div className='message is-danger mt-5'>
			<div className='message-body'>{error}</div>
		</div>
	);
};

export default ErrorDisplay;
