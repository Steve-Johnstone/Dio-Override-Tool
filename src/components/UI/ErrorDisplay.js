import PropTypes from 'prop-types';

const ErrorDisplay = ({ error }) => {
	return (
		<div className='message is-danger mt-5'>
			<div className='message-body'>{error}</div>
		</div>
	);
};

ErrorDisplay.propTypes = {
	error: PropTypes.string,
};

export default ErrorDisplay;
