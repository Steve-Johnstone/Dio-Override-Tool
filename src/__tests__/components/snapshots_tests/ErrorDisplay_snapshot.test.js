import renderer from 'react-test-renderer';
import ErrorDisplay from '../../../components/UI/ErrorDisplay';

describe('ErrorDisplay component', () => {
	it('should display the DOM correctly', () => {
		const tree = renderer
			.create(<ErrorDisplay error='this is an error message' />)
			.toJSON();

		expect(tree).toMatchSnapshot();
	});
});
