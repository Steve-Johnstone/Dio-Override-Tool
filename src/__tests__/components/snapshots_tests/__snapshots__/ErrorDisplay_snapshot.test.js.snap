// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorDisplay component should display the DOM correctly 1`] = `
<div
  className="message is-danger mt-5"
>
  <div
    className="message-body"
  >
    this is an error message
  </div>
</div>
`;
